Modules/Plugins/Armory.lua

```diff
@@ -170,14 +173,17 @@ A.characterSlots = {
   ["SecondaryHandSlot"] = {
     id = 17,
     needsEnchant = true,
-    enchantCondition = {
+    messageCondition = {
       itemType = LE_ITEM_CLASS_WEAPON,
       level = 70,
     },
     needsSocket = false,
     direction = A.enumDirection.LEFT,
   },
 }
+A.lightGreen = { r = 0.07, g = 0.90, b = 0.15 }
+A.darkGreen = { r = 0, g = 0.69, b = 0.11 }
+A.red = { r = 0.94, g = 0.33, b = 0.31 }
```

Comment:
Can you use `HexToRGB` here? https://github.com/Toxicom/toxiui/blob/development/Core/Functions/Table.lua#L127-L130

Reply:
And maybe move these color definitions to L43, where `enumDirection` is defined?